airflow/providers/amazon/aws/operators/sagemaker.py

```diff
@@ -51,9 +51,10 @@ class SageMakerBaseOperator(BaseOperator):
     ui_color: str = "#ededed"
     integer_fields: list[list[Any]] = []
 
-    def __init__(self, *, config: dict, **kwargs):
+    def __init__(self, *, config: dict, aws_conn_id: str, **kwargs):
```

Comment:
```suggestion
    def __init__(self, *, config: dict, aws_conn_id: str = DEFAULT_CONN_ID, **kwargs):
```

Some suggestion which also might fix failed tests